packages/normalizr/src/schemas/__tests__/__snapshots__/Array.test.js.snap

```diff
@@ -969,6 +969,31 @@ Array [
     ],
     undefined,
   ],
+  Array [
```

Comment:
not the best test....as the snapshot needs to be updated based on the extra calls to this.getSchemaAttribute() to format the console warning